tasks/modules/install-modules.yml

```diff
@@ -8,9 +8,11 @@
     - nginx_install_epel_release | bool
 
 - name: (Amazon Linux/RHEL 7) Install GeoIP2 dependency
+  vars:
```

Comment:
See comment on logrotate tasks. Let's do the comparison for this directly in the `when` conditional block.